index.bs

```diff
@@ -4401,6 +4401,11 @@ for {{DOMException}}, a description, and legacy code values.
 
 Note: If an error name is not listed here, please file a bug as indicated at the top of this specification and it will be addressed shortly. Thanks!
 
+Note: Do not confuse {{SyntaxError!!exception}} defined here with ECMAScript's
```

Comment:
Think it would make sense to use the (correct)  `"{{SyntaxError!!exception}}" {{DOMException}}` notation here.